app/views/users/index.html.erb

```diff
@@ -0,0 +1,13 @@
+<h1>Users</h1>
+
+
+
+<ul>
+  <% @user.each do |user| %>
+    <li>
+      <%= link_to user.username, user_path(user) %>
+    </li>
+  <% end %>
+</ul>
```

Comment:
- [ ] Kindly remove logic from your views. Logic includes loops and if statements. This will ensure your project follows the MVC pattern. You can create all your logic inside the helpers. ⬇️ 

Kindly use this article to learn more about this. [Article](http://codefol.io/posts/Where-Do-I-Put-My-Code/)